packages/expo-cli/src/commands/client/index.js

```diff
@@ -249,132 +250,84 @@ export default program => {
     .command('client:install:ios')
     .description('Install the Expo Client for iOS on the simulator')
     .asyncAction(async () => {
-      let currentSdk;
-      const getIosVersion = sdkVersion =>
-        sdkVersion.iosClientVersion || sdkVersion.iosVersion || 'version unknown';
+      const currentSdkConfig = await ClientUpgradeUtils.getExpoSdkConfig(process.cwd());
+      const currentSdkVersion = currentSdkConfig ? currentSdkConfig.sdkVersion : undefined;
```

Comment:
I'm not sure if we should replace this with `ConfigUtils.getExpoSDKVersion`. Is this something we should add now? (I only know about this because it was included in PR 1337 - dont ask...)